homeassistant/components/universal/media_player.py

```diff
@@ -105,8 +107,15 @@
 CONF_CHILDREN = "children"
 CONF_COMMANDS = "commands"
 
-OFF_STATES = [STATE_IDLE, STATE_OFF, STATE_UNAVAILABLE, STATE_UNKNOWN]
-
+STATES_ORDER = [
+    STATE_UNKNOWN,
+    STATE_UNAVAILABLE,
+    STATE_OFF,
+    STATE_ON,
+    STATE_IDLE,
```

Comment:
I think `ON` is more important than `IDLE`. IDLE does nothing for sure, `ON` could be that it's playing.